encord/client_metadata_schema.py

```diff
@@ -0,0 +1,53 @@
+import json
+import uuid
+from typing import Any, Dict, Optional
+from uuid import UUID
+from datetime import datetime
+import encord.orm.client_metadata_schema as orm
+from encord.http.v2.api_client import ApiClient
+
+
+class ClientMetadataSchema:
+
+    def __init__(self, api_client: ApiClient, client_metadata_schema: orm.ClientMetadataSchema | None):
```

Comment:
Unfortunately, we're using only old syntax in the client, as we support python 3.8, so `orm.ClientMetadataSchema | None` => `Optional[ClientMetadataSchema]` or smth